wire-runtime/src/test/proto-kotlin/com/squareup/wire/protos/kotlin/person/Person.kt.java.interop

```diff
@@ -0,0 +1,225 @@
+// Code generated by Wire protocol buffer compiler, do not edit.
+// Source file: person.proto
+package com.squareup.wire.protos.kotlin.person
+
+import com.squareup.wire.EnumAdapter
+import com.squareup.wire.FieldEncoding
+import com.squareup.wire.Message
+import com.squareup.wire.ProtoAdapter
+import com.squareup.wire.ProtoReader
+import com.squareup.wire.ProtoWriter
+import com.squareup.wire.TagHandler
+import com.squareup.wire.WireEnum
+import com.squareup.wire.internal.Internal
+import kotlin.Deprecated
+import kotlin.DeprecationLevel
+import kotlin.Int
+import kotlin.String
+import kotlin.collections.List
+import kotlin.jvm.JvmField
+import okio.ByteString
+
+data class Person(
+    val name: String,
+    val id: Int,
+    val email: String? = null,
+    val phone: List<PhoneNumber> = emptyList(),
+    val unknownFields: ByteString = ByteString.EMPTY
+) : Message<Person, Person.Builder>(ADAPTER, unknownFields) {
+    @Deprecated(
+            message = "Shouldn't be used in Kotlin",
+            level = DeprecationLevel.HIDDEN
+    )
+    override fun newBuilder(): Builder {
+        val builder = Builder()
+        builder.name = name
+        builder.id = id
+        builder.email = email
+        builder.phone = phone
+        builder.addUnknownFields(unknownFields())
+        return builder
+    }
+
+    class Builder : Message.Builder<Person, Builder>() {
+        private var name: String? = null
+
+        private var id: Int? = null
+
+        private var email: String? = null
+
+        private var phone: List<PhoneNumber> = mutableListOf()
+
+        /**
+         * The customer's full name. */
+        fun name(name: String): Builder {
+            this.name = name
+            return this
+        }
+
+        /**
+         * The customer's ID number. */
+        fun id(id: Int): Builder {
+            this.id = id
+            return this
+        }
+
+        /**
+         * Email address for the customer. */
+        fun email(email: String?): Builder {
+            this.email = email
+            return this
+        }
+
+        /**
+         * A list of the customer's phone numbers. */
+        fun phone(phone: List<PhoneNumber>): Builder {
+            Internal.checkElementsNotNull(phone)
+            this.phone = phone
```

Comment:
I am changing the initialization so that it these lists are an instance of  `MutableOnWriteList` which is going to make sure that they are not changed after the first allocation. (the same we do in java gen)